sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/cross/CrossType.scala

```diff
@@ -40,25 +40,23 @@ object CrossType {
 
   object Full extends CrossType {
     def projectDir(crossBase: File, projectType: String): File =
-      crossBase.getAbsoluteFile / projectType
+      crossBase / projectType
```

Comment:
Restating my comment from the old PR:
@gzm0 What was the reason for the absolute paths in the first place?